cli/config/configuration.toml.in

```diff
@@ -177,6 +177,21 @@ path = "@SHIMPATH@"
 # There is no field for this section. The goal is only to be able to
 # specify which type of agent the user wants to use.
 
+[netmon]
+# If enabled, the network monitoring process gets started when the
+# sandbox is created. This allows for the detection of some additional
+# network being added to the existing network namespace, after the
+# sandbox has been created.
+# (default: disabled)
+#enable = true
```

Comment:
nits: as I put a test commit at https://github.com/kata-containers/tests/pull/743, I noticed that it would be better to call this `enable_netmon` instead, so that we can easily grep and replace it in scripts.

Reply:
Of course I can do that and I agree this might help!